ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandRunnerTest.java

```diff
@@ -77,7 +77,7 @@ public void testThread() throws InterruptedException {
 
     Map<CommandId, CommandStatus> statusStore = statementExecutor.getStatuses();
     Assert.assertNotNull(statusStore);
-    Assert.assertEquals(4, statusStore.size());
+//    Assert.assertEquals(4, statusStore.size());
```

Comment:
Do we have a suspicion for why this is flaky on jenkins? We should not disable assertions like this lightly, but instead have a reasonable explanation and --preferably-- a plan to fix.